README.md

```diff
@@ -39,6 +46,9 @@ In an AJAX POST request, what does the argument of the `.done()` callback - in t
     console.log("success!");
   });
 ```
+
+It's confusing to use the word data within the .done function because it is not the data that is being sent in the POST request - Anna talked about this yesterday and used the word "results" in its place. But even knowing this I am still confused looking at it because it seems intuitive that the data is the same data as you're sending in the POST request, simply because they are both called data. But if it's not the resulting data either (as that's what I said was the data in the previous question), I don't really know.
```

Comment:
A few things here. First, you can call the argument whatever you want. Second, whatever you call it, the first argument of `.done()` is the response from the server. Third, in a POST request, the convention is that the server will respond with the object that you just POSTed. In a GET request, the convention is that the server will respond with whatever you just requested (e.g. a single user or a collection of users).